examples/tv-casting-app/tv-casting-common/core/CastingPlayer.cpp

```diff
@@ -243,6 +253,27 @@ CHIP_ERROR CastingPlayer::StopConnecting()
     mTargetCastingPlayer.reset();
     CastingPlayerDiscovery::GetInstance()->ClearCastingPlayersInternal();
 
+    // shouldSendIdentificationDeclarationMessage is false only when StopConnecting() is called by the casting library.
```

Comment:
The implementation should not care about where it is called from, so remove this comment. Add a comment on the call sites to indicate why true / false is being passed

Reply:
Makes sense.